SingularityService/src/main/java/com/hubspot/singularity/data/UsageManager.java

```diff
@@ -95,6 +104,10 @@ private String getCurrentSlaveUsagePath(String slaveId) {
     return ZKPaths.makePath(getSlaveUsagePath(slaveId), CURRENT_USAGE_NODE_KEY);
   }
 
+  private String getSpecificClusterUtilizationPath(long timestamp) {
```

Comment:
why `SpecificClusterUtilization` instead of just `getUsageSummaryPath` or something like that?